docs/declarative-customization/site-design-pnp-provisioning.md

```diff
@@ -172,14 +172,31 @@ Copy the following provisioning template XML to a new file and save the file as
 1. Go to the [Azure Portal](https://portal.azure.com).
 1. Choose **+ Create a resource**.
 1. Search for **Function App** and create a new function app. In the **Storage** field, select **Use existing**, and select the storage account that you created earlier. Set the other values as required.
-1. Currently Function apps are created with the V2 runtime which does not support PowerShell functions. Change the runtime of the function app to V1 by navigating selecting **Platform features**, **Function app settings** and change the Runtime version from **~2** to **~1**.
-1. Create a new function: **Functions** > **New function**.
+1. Navigate within the Function App just created to **Configuration** > **Function runtime settings** and change the runtime version from **~3** to **~1**.
+ 
+    ![Screenshot of the Azure portal with the Function runtime settings screen highlighted](images/pnpprovisioning-runtime-settings.png)
+
+    > [!NOTE]
+    > Function apps based on the runtime version ~3 or ~2 support only PowerShell Core as programming language for the functions.

```

Comment:
Rephrase for readability "support only" > "only support" and "functions" => "Function apps"